.github/workflows/cypress.yaml

```
@@ -81,58 +77,24 @@ jobs:
+      - name: Install Concurrently
+        run: npm install -g concurrently
+
+      - name: Fix Permissions for Test Files
+        run: chmod -R +rwx cypress/e2e
+
+      - name: Debug Test File Discovery
+        run: find cypress/e2e -type f -name "*.cy.ts"
 
-      - name: Upload cypress videos 📹
+      - name: Run Cypress Tests in Parallel
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2016:info:3:46: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)